app/views/govuk_publishing_components/components/_layout_for_public.html.erb

```diff
@@ -43,7 +43,7 @@
       search: true,
     } %>
 
-    <main class="<%= "govuk-width-container" unless full_width %>" id="content">
+    <main class="<%= "govuk-width-container govuk-main-wrapper" unless full_width %>" id="content">
```

Comment:
In the Design System docs these two classes are applied on separate elements, is it okay to combine them like this?

Reply:
I'd leave `govuk-width-container` on `main` as that seems to be the class we need to achieve the width constraints. 
The other `govuk-main-wrapper` class seems to add extra padding which we might not always be desirable or needed (it might become a head ache if we end up having to override it often. I reckon we can always add padding/margins to the inner elements instead, if necessary?).  

But then again I haven't got full context of why this was done the way it was.  Please let me know your thoughts on this?

Reply:
I think we need both classes but I don't know if it's okay to combine them like this. Perhaps @36degrees can illuminate?

Reply:
@andysellick I've checked with DS team and they have confirmed that this is not an appropriate use of these classes, so good shout there.

> They should be separate, because any phase banner, breadcrumbs or back link should sit inside the width container but outside of the main wrapper. There’s some guidance around this on https://design-system.service.gov.uk/styles/layout/#page-wrappers which might be useful.

On the basis of the quoted comment above, I've also added a new block that sits outside `main.govuk-main-wrapper`, but inside `.govuk-width-container`.  This introduces the ability to pass through breadcrumbs/backlink/any other element that should exist as a sibling of `.govuk-main-wrapper`.

Please feel free to flag this conversation as resolved if all looks good to you.